src/Commands/Domain/ListCommand.php

```diff
@@ -29,6 +29,7 @@ class ListCommand extends TerminusCommand implements SiteAwareInterface
      * @field-labels
      *     id: Domain/ID
      *     type: Type
+     *     primary: Is Primary
```

Comment:
I don't know if it's considered a b/c break to add a column to the default output. You could define `@default-fields: id, type, ... etc.` to maintain the same output. If you did that, though, then no one would see `primary` unless it was explicitly requested via `--fields`. I think it's probably better this way.